models/admin.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require("mongoose");
+
+const { Schema } = mongoose;
+
+/**
+ * Admin Model
+ */
+module.exports = mongoose.model(
+  "Admin",
+  new Schema({
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    email: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    phone: {
+      type: String,
+      required: true,
+    },
+    password: {
+      type: String,
+      required: true,
+    },
```

Comment:
Since we are using Firebase for authentication now, I don't think we need to save passwords.